Speech/src/V1/SpeechClient.php

```diff
@@ -30,13 +30,87 @@
 
 namespace Google\Cloud\Speech\V1;
 
+use Generator;
+use Google\ApiCore\ApiException;
+use Google\ApiCore\OperationResponse;
+use Google\Cloud\Speech\SpeechHelpersTrait;
 use Google\Cloud\Speech\V1\Gapic\SpeechGapicClient;
 
 /**
  * {@inheritdoc}
  */
 class SpeechClient extends SpeechGapicClient
 {
-    // This class is intentionally empty, and is intended to hold manual
-    // additions to the generated {@see SpeechClientImpl} class.
+    use SpeechHelpersTrait;
+
+    /**
+     * Helper method to create a RecognitionAudio object from audio data.
+     *
+     * @param resource|string|RecognitionAudio $audio *Required* The audio data to be recognized. This can be a RecognitionAudio
+     *                                                object, a Google Cloud Storage URI, a resource object, or a string of bytes.
+     * @return RecognitionAudio
+     */
+    public function createRecognitionAudio($audio)
+    {
+        return $this->createRecognitionAudioHelper(RecognitionAudio::class, $audio);
+    }
+
+    /**
+     * Helper method to create a stream of StreamingRecognizeRequest objects from audio data.
+     *
+     * @param iterable|resource|string      $audio *Required* The audio data to be converted into a stream of requests. This
+     *                                             can be a resource, a string of bytes, or an iterable of
+     *                                             StreamingRecognizeRequest[] or string[].
+     * @return StreamingRecognizeRequest[]
+     */
+    public function createStreamingRequests($audio)
+    {
+        return $this->createStreamingRequestsHelper(StreamingRecognizeRequest::class, $audio);
+    }
+
+    /**
+     * Performs speech recognition on a stream of audio data. This method is only available via
+     * the gRPC API (not REST).
+     *
+     * Example:
+     * ```
+     * use Google\Cloud\Speech\V1\RecognitionConfig_AudioEncoding;
+     * use Google\Cloud\Speech\V1\RecognitionConfig;
+     * use Google\Cloud\Speech\V1\StreamingRecognitionConfig;
+     *
+     * $recognitionConfig = new RecognitionConfig();
+     * $recognitionConfig->setEncoding(RecognitionConfig_AudioEncoding::FLAC);
+     * $recognitionConfig->setSampleRateHertz(44100);
+     * $recognitionConfig->setLanguageCode('en-US');
+     * $config = new StreamingRecognitionConfig();
+     * $config->setConfig($recognitionConfig);
+     *
+     * $audioResource = fopen('path/to/audio.flac', 'r');
+     *
+     * $responseStream = $speechClient->recognizeAudioStream($config, $audioResource);
```

Comment:
Since we updated this to not return a bidi stream and instead the iterable of recognize responses, could we call this `$responses`?